packages/template-retail-react-app/app/static/translations/compiled/en-GB.json

```diff
@@ -2233,6 +2233,12 @@
       "value": "Apple"
     }
   ],
+  "login_form.button.back": [
+    {
+      "type": 0,
+      "value": "Back"
```

Comment:
sorry @jeremy-jung1 can we change this to `Back to Sign In Options` to be consistent with the checkout login page? This is also something we agreed with Trevor on [here](https://salesforce-internal.slack.com/archives/C055P19CF5Z/p1736287444969459?thread_ts=1736190958.412369&cid=C055P19CF5Z)
<img width="1580" alt="Screenshot 2025-01-17 at 1 35 42 PM" src="https://github.com/user-attachments/assets/33b370ab-f615-452c-8013-c1a04849dda7" />